pkg/cmd/workspace/code.go

```diff
@@ -52,7 +54,21 @@ var CodeCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		if len(args) == 0 {
+		// Check if the user has provided a workspace name
+		if len(args) > 0 {
+			// Validate if the argument is a valid workspace name
+			if isInvalidWorkspaceName(args[0]) {
```

Comment:
I don't think this case is necessary and it makes us maintain the workspace name validity in more than 1 place.

Better handling of the API error should be enough.

Reply:
Sorry for this. Also resolved it now